scenes/cslib_math.txt

```diff
@@ -1,3 +1,5 @@
+*bug "cslib_math is a cslib module and must not be accessed directly"
```

Comment:
Sorry, this is my mistake, I know I've done it in other modules, but *bug doesn't need the quotes.
I'll go through and change the others when I get rid of/replace assert.